etna/search/tests/test_views.py

```diff
@@ -772,13 +773,6 @@ def test_current_bucket(self):
                     is_current=True,
                     results=None,
                 ),
-                Bucket(
-                    key="highlight",
-                    label="Highlights",
-                    result_count=1,
-                    is_current=False,
-                    results=None,
-                ),
```

Comment:
There is some code around website highlight [WebsiteSearchView](https://github.com/nationalarchives/ds-wagtail/blob/9d7bdb78edb9000adcc67bdc676102e9decc001a/etna/search/views.py#L723-L724),  and unit tests are skipped
Would it be Ok to then comment out those bits which are removed?

Reply:
Yes, that's a good shout. Implemented in [c92c7a2](https://github.com/nationalarchives/ds-wagtail/pull/775/commits/c92c7a272164a005c395fe7a5d91123ed25e10e7)